app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id', inverse_of: :posts
+  has_many :comments, class_name: 'Comment', foreign_key: 'post_id', inverse_of: :post, dependent: :destroy
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id', inverse_of: :post, dependent: :destroy
 end
```

Comment:
- [ ] [REQUIRED] Let's also ensure that the Post model has a method that updates the posts counter for a user. It will be nice if you can use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `update_post_counter` method every time a new post has been saved to database,then make the `update_post_counter` method private.
- [ ] [REQUIRED] Kindly ensure that the Post model has a method that returns the 5 most recent comments for a given post.